src/openforms/utils/email.py

```diff
@@ -25,7 +25,11 @@ def send_mail_plus(
         fail_silently=fail_silently,
     )
     mail = EmailMultiAlternatives(
-        subject, message, from_email, recipient_list, connection=connection
+        subject.replace("\n", ""),
```

Comment:
There is an issue for this https://github.com/open-formulieren/open-forms/issues/964 but it's just something I did to get the confirmation emails working.  I think we can probably remove this.